packages/kit/src/views/Market/MarketHomeV2/layouts/DesktopLayout.tsx

```
@@ -7,6 +7,8 @@ import {
   useTabContainerWidth,
 } from '@onekeyhq/components';
 import platformEnv from '@onekeyhq/shared/src/platformEnv';
+import { useRouteIsFocused} from '@onekeyhq/kit/src/hooks/useRouteIsFocused';
+
```

ESLint

[error] Replace `}·from·'@onekeyhq/kit/src/hooks/useRouteIsFocused';⏎` with `·}·from·'@onekeyhq/kit/src/hooks/useRouteIsFocused';`

(prettier/prettier)